rho/inventory_scan.py

```diff
@@ -208,6 +208,14 @@ def inventory_scan(hosts_yml_path, facts_to_collect, report_path,
         postprocessing.handle_systemid(facts_to_collect, this_host)
         postprocessing.handle_redhat_packages(facts_to_collect, this_host)
         postprocessing.escape_characters(this_host)
+        this_host.update(postprocessing.generate_eap_summary(this_host))
+
+        # After all of the facts have been generated, remove -mr facts
+        # which were for machine use only.
+        keys = list(this_host.keys())
+        for key in keys:
+            if key[-3:] == '-mr':
+                del this_host[key]
```

Comment:
I hesitate to remove all these facts as we lose some value that we may get from them. Like versions or deploy dates. May want to circle back with Mark.

Reply:
Overall thought: I don't really like the MR concept. I think the "right" thing to do is to only have machine-readable facts until the very end, and then translate them into user-readable strings. But I also wanted to get this PR done today, so I implemented this as a hackish-thing.

To your point, as far as I know the MR facts have the same information that is in the user-readable facts that we're keeping around. The difference is that in the user-readable facts, it's already been string-formatted.